### DISCUSSION TITLE
How to change shortcuts?

### DISCUSSION THREAD

**Discussion by Devious3901** [audacious-media-player/audacious#74] [Q&A]
How to change shortcuts? no on settings menu
How i can make is?

**Comment by radioactiveman** [comment]
Unfortunately Audacious does not support changing shortcuts. Neither for the Qt nor the GTK interface.

**Reply by itt533** [reply]
are they planning to implement it?

**Reply by hifron** [reply]
there is **plugin for global shortcuts** in Services/plugins... and that could be enabled via XF86Media keys keyboard or set up also default media keys in (**gnome-control-center(with ->)**) or also in **dconf-editor**(_search in bar for media_), but **mpris** plugin is required in Services/plugins

there is also config in .config/audacious to manually set up keys, keyboard keys could be shown in **xev** bash program... or somehow set in keyboard control center if present by vendor.

but problem is default audacious settings and some like default keys can't be changed currently it seems, only additional setup possible...

**Reply by hifron** [reply]
There is also possible to have cron [to change something](https://github.com/keshavbhatt/BingWall/issues/47)... currently somehow cumbercome
